pyproject.toml

```diff
@@ -49,6 +49,12 @@ universal = 0
 profile = "black"
 line_length = 120
 
+[tool.pytest.ini_options]
```

Comment:
now preparation only: warnings collected much better, but non-stable event loop  tests and fully incompatible `test_vterm`